src/icons/fileIcons.ts

```diff
@@ -1163,7 +1163,7 @@ export const fileIcons: FileIcons = {
       ],
     },
     { name: 'apollo', fileNames: ['apollo.config.js'] },
-    { name: 'nodemon', fileNames: ['nodemon.json', 'nodemon-debug.json'] },
+    { name: 'nodemon', fileNames: ['nodemon.json', 'nodemon-debug.json', '.node-dev.json'] },
```

Comment:
Please correct me if I got it wrong but as far as I've understood the project around the file `.node-dev.json` it is similar to Nodemon, but not the same or not related to the project. So I think it's not so good to apply the Nodemon logo to a file which is part of another tool that does something similary as Nodemon but is not really related to the Nodemon project itself.

Reply:
Yep, you're right!
So, we should duplicate the icon?

Reply:
It's not a naming issue, it's the fact that we're using the nodemon icon for this file which is not correct. It's not related to nodemon as far as I understood it. So the only solution here is to create a complete new icon for files like `.node-dev.json`.

Reply:
So, i guess we should close this PR and make an icon request, hum?

Reply:
Yes, I think that would be the best solution :)